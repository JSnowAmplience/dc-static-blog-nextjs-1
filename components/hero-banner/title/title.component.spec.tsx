import renderer from 'react-test-renderer';
import HeroBannerTitle from './title.component';

describe('HeroBannerTitle', () => {
  test('renders hero banner title', async () => {
    const props = {
      title: 'Test title'
    };
    const component = await renderer.create(<HeroBannerTitle {...props} />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
