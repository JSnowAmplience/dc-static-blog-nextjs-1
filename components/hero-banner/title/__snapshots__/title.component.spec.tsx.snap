// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroBannerTitle renders hero banner title 1`] = `
<section
  className="jsx-2086237504"
>
  <h1
    className="jsx-2086237504"
  >
    Test title
  </h1>
</section>
`;
